Ginger/Ginger/WindowExplorer/Common/ControlActionsPage.xaml.cs

```diff
@@ -157,12 +157,16 @@ private void InitActionsGrid()
 
         private void AddToPageList()
         {
-            ControlActionsPage tempPage = App.PageList.FirstOrDefault(Page => Page is ControlActionsPage) as ControlActionsPage;
-            if (tempPage == null) App.PageList.Add(this);
-            else
+            if (App.PageList != null)
             {
-                App.PageList.Remove(tempPage);
-                App.PageList.Add(this);
+                ControlActionsPage tempPage = App.PageList.FirstOrDefault(Page => Page is ControlActionsPage) as ControlActionsPage;
```

Comment:
but why App.PageList comes null now while before it was not happening?

Reply:
In currernt branch, set value statement has removed for App.PageList. 
Previously we were setting the value in App.PageList from MainWindow.xaml.cs file. now that reference has been removed. 
![image](https://user-images.githubusercontent.com/41137863/46716893-5c880400-cc83-11e8-9110-07bb11d6809d.png)